### ISSUE TITLE
ActiveX filtering prevents download of Screenleap app in IE

### ISSUE THREAD

**Issue by LeahW** [Screenleap/support#18]
Impact: New Win native app downloads from using Internet Explorer with ActiveX filtering turned on

STR:
1. In Internet Explorer, click on Tools and check the box for 'ActiveX filtering'
2. Go to www.screenleap.com using Internet Explorer
3. Click on the green 'Share your screen now' button to start download
4. See 'Screenleap is starting . . . ' message but eventually re-directed to 'We were not able to start your screen share' error.

Workaround: Turn off ActiveX filtering or add www.screenleap.com to trusted sites. Also, if you can download the app using another browser and install it, you are able to launch screen shares from Internet Explorer.

**Comment by tuyentruong** [comment]
@LeahW Do you know if this happens in production?

**Comment by tuyentruong** [comment]
@LeahW Also, what version of IE are you using?

**Comment by LeahW** [comment]
It was not happening last night when I entered www.stage.screenleap.com and was getting redirected to production. But it does happen now, on stage, when redirected to prod, and when going straight to production.
IE 11.0.9600.18015

**Comment by tuyentruong** [comment]
@LeahW This works for me on IE10 and IE11 on staging. Can you retest to see if it's still a problem for you?

**Comment by LeahW** [comment]
I figured it out; even though my Security level was set to Medium, I had 'Active X filtering' checked in Tools. Once I unchecked it, the icon immediately stopped spinning and the download started.

![activexinie](https://cloud.githubusercontent.com/assets/9301423/10365601/98a1017e-6d92-11e5-96e8-f11d12f5c54c.png)

Perhaps we could add a suggestion to the IE 'We were not able to start your screen share' error page that user turn off 'Active X filtering' and click the link to restart download?
